bin/hubot.cmd

```diff
@@ -0,0 +1,7 @@
+@echo off
+
+call npm install
```

Comment:
Why npm and not yarn?

Reply:
This file was auto-generated when running the hubot generator so I didn't touch it. Changing it to `yarn` sgtm.